app/lib/document_types/consent_form.rb

```diff
@@ -0,0 +1,19 @@
+module DocumentTypes
+  class ConsentForm < DocumentType
+    class << self
+      def relevant_to?(intake)
+        # This is not used in a flow or to determine relevant types for an intake
+        # and will likely be deprecated after we implement client authentication
+        false
+      end
+
+      def key
+        "Consent Form"
```

Comment:
If this is the 14446 Form, might make sense to make the key or display name make that really clear by including "f14446" in the key and/or the creation name.